lib/udev/rules.d/90-revpi-bluetooth.rules

```diff
@@ -0,0 +1,5 @@
+# Revolution Pi bluetooth module: start the hciuart service
+ACTION=="add", DEVPATH=="*/usb1/1-1/1-1.5/1-1.5:1.0/*", TAG+="systemd", ENV{SYSTEMD_WANTS}+="revpi-hciuart@%k.service"
+
+# Revolution Pi bluetooth module: start the revpi-bthelper service
+ACTION=="add", SUBSYSTEM=="bluetooth", KERNEL=="hci[0-9]", TAG+="systemd", ENV{SYSTEMD_WANTS}+="revpi-bthelper@%k.service"
```

Comment:
Both rules have the same routing:  rule matching --> systemd service --> shell script

Reply:
And what do you want to tell me with this?

Reply:
I just try to explain how the shell scripts get executed. 
If you can easily understand the  *.rules,  *.services, shell script, you can simply ignore this.